### DISCUSSION TITLE
Feature: Default user permissions for all Sections

### DISCUSSION THREAD

**Discussion by missmatsuko** [craftcms/cms#8614] [Ideas]
### Description
I saw some issues about making more granular permissions settings, but could we consider more general ones as well?

For example, I think that having default permissions for user groups for all Sections could be useful.

In all the sites I've built so far, I have wanted users to have access to the same permissions on every Section.

e.g. Editors should be able to create and delete their own entries in all Sections. Publishers should have all abilities for all Sections.

Currently I have to remember to go through all the user groups whenever I add a new Section, and make the permissions match the other Sections.

I often forget to update permissions when I add new Sections to an existing site because I am able to see the new Section right away in my admin account.

**Comment by brandonkelly** [comment]
We can look into this alongside #1646.
